kotlin/build.gradle.kts

```diff
@@ -13,25 +13,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import org.jetbrains.dokka.gradle.DokkaTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jlleitschuh.gradle.ktlint.KtlintExtension
+import org.jlleitschuh.gradle.ktlint.reporter.ReporterType
+
 buildscript {
   dependencies {
-    classpath Dep.get("android_gradle_plugin")
-    classpath Dep.get("detekt")
-    classpath Dep.get("dokka")
-    classpath Dep.get("jmh.gradlePlugin")
-    classpath Dep.get("kotlin.binaryCompatibilityValidatorPlugin")
-    classpath Dep.get("kotlin.gradlePlugin")
-    classpath Dep.get("kotlin.serialization.gradlePlugin")
-    classpath Dep.get("ktlint")
-    classpath Dep.get("mavenPublish")
+    classpath(get("android_gradle_plugin"))
```

Comment:
Will these become field references instead of `get()` calls in a follow up?

Reply:
Already a todo to fix in this PR https://github.com/square/workflow/pull/1073#issuecomment-607553854